pages/about/groups/agwg.md

```diff
@@ -33,67 +34,80 @@ Some information on this page is also shown on, and may be more current in, the
 {% include_cached toc.html type="end" %}
 {:/}
 
-## Announcements
+## Admin home page
```

Comment:
This is the one bit I don't think works. The language isn't quite right and it doesn't really communicate what the page is or does. It also doesn't seem that useful here. 

My view would be remove this section entirely. Access to the W3 group pages can be achieved via links below that are much more descriptive in purpose.

Reply:
Done. 
I agree. It's not a quick link that users to this page are likely to be looking for.